Rendering/Core/vtkLabeledContourMapper.cxx

```diff
@@ -603,6 +603,7 @@ bool vtkLabeledContourMapper::PrepareRender(vtkRenderer *ren, vtkActor *act)
       continue;
       }
     metric.Value = scalars->GetComponent(ids[0], 0);
+    metric.Value = std::fabs(metric.Value) > 1e-6 ? metric.Value : 0.0;
```

Comment:
@durack1 I don't think so. 

If the value is **greater** than a negligible amount (`1e-6`) then keep the value or reset it to `0.0`